src/Microsoft.DotNet.XHarness.iOS.Shared/Logging/AggregatedLog.cs

```diff
@@ -3,15 +3,23 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Generic;
 using System.IO;
 using Microsoft.DotNet.XHarness.Common.Logging;
 
+#nullable enable
 namespace Microsoft.DotNet.XHarness.iOS.Shared.Logging
 {
 
     public abstract partial class Log
     {
 
+        public static ILog CreateAggregatedLogWithDefault(ILog defaultLog, params ILog[] logs)
```

Comment:
I think it would maybe make sense to call it `AppendLog`? And use the base one as the default one?

Reply:
I was thinking about how to fix this mess. I have a plan. I'd say, lets merge this, then I'll create a PR with a proper fix and will move all the stuff. Specially since we do not yet consume this from xamarin-macios, we can change it and make it properly.